Date.php

```diff
@@ -481,11 +481,10 @@ class Date
      *                                    (Defaults to
      *                                    {@link DATE_FORMAT_ISO}.)
      *
-     * @return   void
      * @access   public
      * @see      Date::setDate()
      */
-    function Date($date = null, $options = null)
+    function __construct($date = null, $options = null)
```

Comment:
This should add the `public` to let developer know this constructor is `public`.